lizzy/metrics.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+import logging
+import os
+
+from lizzy import config
+from metricz import MetricWriter
+
+server = None
+logger = logging.getLogger('lizzy.metrics')  # pylint: disable=invalid-name
+
+
+def push(key, value):
+    if config.kairosdb_url:
+        global server
+        if not server:
+            server = MetricWriter(kairosdb_url=config.kairosdb_url,
+                                  fail_silently=True,
+                                  timeout=4)
+        try:
+            server.write_metric(
+                '{}.lizzy.{}'.format(config.metrics_prefix, key),
```

Comment:
The general format is `bus.deployctl.update.projects_created`

Basically `<team>.<project-name>.<key>` (in small letters). It just keeps them consistent with rest of the kairosdb keys.